airflow/contrib/operators/grpc_operator.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.contrib.hooks.grpc_hook import GrpcHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class GrpcOperator(BaseOperator):
+    """
+    Calls a gRPC endpoint to execute an action
+    """
+    template_fields = ('stub_class', 'call_func', 'data')
+    # hook dispatcher dict, dispatch based on auth_type other than the ones
+    # support in the base hook
+    hook_dispatchers = {}
+
+    @apply_defaults
+    def __init__(self,
+                 stub_class,
+                 call_func,
+                 grpc_conn_id="grpc_default",
+                 data=None,
+                 auth_type="NO_AUTH",
+                 interceptors=None,
+                 custom_connection_func=None,
+                 streaming=False,
+                 response_callback=None,
+                 log_response=False,
+                 *args, **kwargs):
+        """
+        :param stub_class: The stub client to use for this gRPC call
+        :type stub_class: gRPC stub class generated from proto file
```

Comment:
This is not the correct syntax. You can not add documentation to the class and constructor at the same time. You must select only one option, otherwise one of the options will be ignored in the process of building documentation.

Reply:
Thanks. Let me fix it